Makefile

```diff
@@ -0,0 +1,10 @@
+SHELL := /bin/bash
+
+# List of targets the `readme` target should call before generating the readme
+export README_DEPS ?= docs/targets.md docs/terraform.md
```

Comment:
why the Markdown docs are targets or dependencies?

Reply:
this is classic make. In fact, historically, targets were usually files or globs. 

https://github.com/cloudposse/build-harness/blob/master/modules/docs/Makefile#L5-L18

![image](https://user-images.githubusercontent.com/52489/41933201-0ae75f04-7938-11e8-96e6-66c15685a3c6.png)